WooCommerce/Resources/de.lproj/Localizable.strings

```diff
@@ -1156,9 +1156,6 @@
 /* Title of the view for editing the coupon description. */
 "Coupon Description" = "Beschreibung des Gutscheins";
 
-/* Field in the view for adding or editing a coupon. */
-"Coupon Expiry Date" = "Coupon Expiry Date";
-
```

Comment:
This is not missing in the legacy automation counter part, but it is a legit missing translation at the time I run the automation:

<img width="917" alt="image" src="https://user-images.githubusercontent.com/1218433/170183617-3783deaf-3e02-42e9-a5e3-edb317cfe418.png">